File: api/health.py
from http.server import BaseHTTPRequestHandler
import json
from datetime import datetime

class handler(BaseHTTPRequestHandler):
    def do_GET(self):
        self.send_response(200)
        self.send_header('Content-Type', 'application/json')
        self.end_headers()
        self.wfile.write(json.dumps({
            "status": "healthy",
            "timestamp": datetime.now().isoformat(),
            "service": "Gatekeeper AI API",
            "version": "1.0"
        }).encode())

File: api/health.js
// api/health.js
// Gatekeeper AI – Health Check Endpoint

export default function handler(req, res) {
    res.setHeader('Access-Control-Allow-Origin', '*');
    res.setHeader('Content-Type', 'application/json');

    const health = {
        status: 'healthy',
        timestamp: new Date().toISOString(),
        uptime: process.uptime(),
        services: {
            api: 'running',
            database: 'connected',
            whatsapp: 'pending',
            server: 'online'
        },
        version: '1.0.0'
    };

    res.status(200).json(health);
}
